sdk/src/types/block/output/account.rs

```diff
@@ -491,6 +491,14 @@ impl StateTransitionVerifier for AccountOutput {
     }
 }
 
+impl WorkScore for AccountOutput {
+    fn work_score(&self, work_score_params: WorkScoreParameters) -> u32 {
+        let features_score = self.features().work_score(work_score_params);
+        let immutable_features_score = self.immutable_features().work_score(work_score_params);
+        work_score_params.output() + features_score + immutable_features_score
```

Comment:
Would just add them directly without defining a variable first

Reply:
yeah, that's the same comment basically as Thibault's :trollface: Changed it.
(Did it like that so that it's easier to compare with the TIP)